File: ballast-client/npm/src/signalr/signalr-client-options.ts
export interface ISignalRClientOptions {
    serverUrl: string;
    clientId: string;
}

export class SignalRClientOptions implements ISignalRClientOptions {
    public serverUrl: string;
    public clientId: string;
    public constructor(serverUrl: string, clientId: string) {
        this.serverUrl = serverUrl;
        this.clientId = clientId;
    }
}

File: ballast-client/dotnet/src/signalr/SignalRClientOptions.cs
using System;

namespace Ballast.Client.SignalR 
{
    public interface ISignalRClientOptions
    { 
        string ServerUrl { get; set; }
        Guid ClientId { get; set; }
    }

    public class SignalRClientOptions : ISignalRClientOptions 
    { 

        public string ServerUrl { get; set; }
        public Guid ClientId { get; set; }

        public SignalRClientOptions(string serverUrl, Guid clientId) 
        { 
            ServerUrl = serverUrl;
            ClientId = clientId;
        }

    }
}